src/utils/crypto.ts

```diff
@@ -8,7 +8,7 @@ type Algorithm = {
   alias: string
 }
 
-type Data = string | boolean | number | object | ArrayBufferView | ArrayBuffer | ReadableStream
+type Data = string | boolean | number | object | ArrayBufferView | ArrayBuffer
```

Comment:
```suggestion
type Data = string | boolean | number | object
```

I think we can remove `ArrayBuffer`, `ArrayBufferView`, etc. since they are subtypes of `object`.

Reply:
Thanks for pointing that out! I see.
It's true that it allows any `object` (including those that cannot be `JSON.stringify` ), not just `ArrayBufferView` or `ArrayBuffer`.

Semantically, I think the following changes are appropriate.

```diff
diff --git a/src/utils/crypto.ts b/src/utils/crypto.ts
index 9ffb8310..68fcf8ca 100644
--- a/src/utils/crypto.ts
+++ b/src/utils/crypto.ts
@@ -3,12 +3,14 @@
  * Crypto utility.
  */
 
+import type { JSONValue } from './types'
+
 type Algorithm = {
   name: string
   alias: string
 }
 
-type Data = string | boolean | number | object | ArrayBufferView | ArrayBuffer
+type Data = string | boolean | number | JSONValue | ArrayBufferView | ArrayBuffer
 
 export const sha256 = async (data: Data): Promise<string | null> => {
   const algorithm: Algorithm = { name: 'SHA-256', alias: 'sha256' }
```

However, there is a slight possibility (I think it's very slight) that it could affect compatibility, so I think it would be better to consider it in a separate pull request.

Reply:
I agree.